src/content/glossary/d1.yaml

```
@@ -1,6 +1,21 @@
 ---
 productName: D1
 entries:
+  - term: primary database instance
+    general_definition: |-
+      the primary database instance is the original instance of a database. This database instance only exists in one location in the world.
+  - term: read replica
+    general_definition: |-
+      a read replica is an “almost up-to-date” copy of the primary database instance which only serve read requests. There may be multiple read replicas for a single primary database instance.
```

"Almost up-to-date" is weird. 
```suggestion
      a read replica is a eventually replicated copy of the primary database instance which only serve read requests. There may be multiple read replicas for a single primary database instance.
```